db/migrate/20220325091605_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] As per the given ERD, the **foreign key** is supposed to be _AuthorId_ instead of _user_. Kindly ensure to improve upon this aspect.